### PR TITLE
3-layers Move SocialNetworkImageService

### PR DESCRIPTION
**Pull request by olenapetryshak**
https://github.com/ita-social-projects/GreenCity/issues/1614

### CODE REVIEW

File: service-api/src/main/java/greencity/dto/socialnetwork/SocialNetworkImageVO.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@Builder
+public class SocialNetworkImageVO {
+    private Long id;
+    String imagePath;
+    String hostPath;
```
Comment (vterlyha) [inline service-api/src/main/java/greencity/dto/socialnetwork/SocialNetworkImageVO.java:14]: should be private

File: service/pom.xml [hunk 2]
```diff
@@ -82,6 +87,16 @@
             <version>1.105.0</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.6</version>
```
Comment (vterlyha) [inline service/pom.xml:93]: Please move version to properties

File: service/pom.xml [hunk 3]
```diff
@@ -82,6 +87,16 @@
...
+        </dependency>
+        <dependency>
+            <groupId>commons-fileupload</groupId>
+            <artifactId>commons-fileupload</artifactId>
+            <version>1.3</version>
```
Comment (vterlyha) [inline service/pom.xml:98]: same here

File: service/src/main/java/greencity/service/SocialNetworkImageServiceImpl.java [hunk 4]
```diff
@@ -57,13 +54,13 @@
...
-    public SocialNetworkImage getSocialNetworkImageByUrl(String url) {
+    public SocialNetworkImageVO getSocialNetworkImageByUrl(String url) {
         try {
             URL checkUrl = new URL(url);
-            Optional<SocialNetworkImage> optionalSocialNetworkImage =
+            Optional<SocialNetworkImageVO> optionalSocialNetworkImageVO =
                 findByHostPath(checkUrl.getHost());
-            if (optionalSocialNetworkImage.isPresent()) {
-                return optionalSocialNetworkImage.get();
+            if (optionalSocialNetworkImageVO.isPresent()) {
```
Comment (vterlyha) [inline service/src/main/java/greencity/service/SocialNetworkImageServiceImpl.java:62]: Use ternary expression ove if/else statement. It is more readable

File: service-api/src/main/java/greencity/dto/socialnetwork/SocialNetworkImageVO.java (hunk 1 above)
Comment (olenapetryshak) [reply service-api/src/main/java/greencity/dto/socialnetwork/SocialNetworkImageVO.java]: done

File: service/pom.xml (hunk 2 above)
Comment (olenapetryshak) [reply service/pom.xml]: done

File: service/pom.xml (hunk 3 above)
Comment (olenapetryshak) [reply service/pom.xml]: done

File: service/src/main/java/greencity/service/SocialNetworkImageServiceImpl.java (hunk 4 above)
Comment (olenapetryshak) [reply service/src/main/java/greencity/service/SocialNetworkImageServiceImpl.java]: done

**Review by Markiyan000 — APPROVED** [review]

**Review by IvanBegar — APPROVED** [review]

**Review by TarasDovganyuk — APPROVED** [review]

**Pull request merged**